tutorials/udp/mdns-server/main.c

```diff
@@ -12,6 +12,7 @@ int main(void) {
 
   // Desired name must NOT have any dots in it, nor a domain
   c = mg_mdns_listen(&mgr, "Mongoose");  // Start mDNS server
+  if (!c) return 1;
```

Comment:
`c` is not a boolean or bit, so we should check for `== NULL` (it is less non-elegant when checking for !=0 because there is no '!' shouting)